recipes-scanners/clamav/clamav_1.4.1.bb

```diff
@@ -0,0 +1,226 @@
+SUMMARY = "ClamAV anti-virus utility for Unix - command-line interface"
+DESCRIPTION = "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats."
+HOMEPAGE = "http://www.clamav.net/index.html"
+SECTION = "security"
+LICENSE = "LGPL-2.1-only"
+
+DEPENDS = "glibc llvm libtool db openssl zlib curl libxml2 bison pcre2 json-c libcheck rust-native cargo-native libmspack"
+ 
+COMPATIBLE_HOST:libc-musl:class-target = "null"
+
+LIC_FILES_CHKSUM = "file://COPYING.txt;beginline=2;endline=3;md5=f7029fbbc5898b273d5902896f7bbe17"
+
+# July 30th, 2022
+SRCREV = "e81cc6f4271fbdb2e40a9734e06bb507441d6e09"
```

Comment:
What commit is this ???

And why aren't we using release 1.4.3 - which is the latest LTS release?